src/main/webapp/javascript/main.js

```diff
@@ -28,4 +27,27 @@ function logout() {
 }
 function goMyProfile() {
     location.href = "/profile.jsp";
+}
+function join(button) {
+    var data = {};
+    var places = $(button).closest('tr').find('.table_places').text() - 1;
+    var tripId = $(button).closest('tr').find('.table_id').text();
+    var id = $(button).closest('td').parent().index();
```

Comment:
This code will easily break if something changes in DOM structure. Can you pass these variables as parameters to the `join()` method?`